dispatcher/subscriber.go

```diff
@@ -0,0 +1,23 @@
+package dispatcher
+
+import (
+	"context"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-proto/golang/iotexrpc"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+// Subscriber is the dispatcher subscriber interface
+type Subscriber interface {
+	ReportFullness(context.Context, iotexrpc.MessageType, float32)
+	HandleAction(context.Context, *iotextypes.Action) error
+	HandleBlock(context.Context, string, *iotextypes.Block) error
+	HandleSyncRequest(context.Context, peer.AddrInfo, *iotexrpc.BlockSync) error
+	HandleConsensusMsg(*iotextypes.ConsensusMessage) error
+	HandleNodeInfoRequest(context.Context, peer.AddrInfo, *iotextypes.NodeInfoRequest) error
+	HandleNodeInfo(context.Context, string, *iotextypes.NodeInfo) error
+	HandleActionRequest(ctx context.Context, peer peer.AddrInfo, actHash hash.Hash256) error
```

Comment:
better name to indicate for special type of action (blob so far)

Reply:
it is generic mechanism for action sync, not limited only for blob, but we just use blob on it